app-misc/anki/files/23.12.1/remove-yarn-dep.patch

```diff
@@ -0,0 +1,68 @@
+This is an adapted patch from the games/anki FreeBSD port.
```

Comment:
Could you add links to where you got it from/the inspiration? In general, the more references the better, even if you adapted it or only used it as inspiration.